.github/workflows/reusable_build.yml

```diff
@@ -50,7 +50,13 @@ jobs:
           # forbids interactive configuration phase. Otherwise the CI gets
           # stuck from time to time on `apt-get update`.
           DEBIAN_FRONTEND: noninteractive
-        run: make -f .gitlab.mk package
+        run: |
+          # If a packpack container with volumes starts by not a root user,
+          # it's needed to create the volume dir first to avoid read/write
+          # permission issues.
+          rm -rf ${{ env.VARDIR }}
+          mkdir -p ${{ env.VARDIR }}
+          make -f .gitlab.mk package
```

Comment:
I think we shouldn't run testing of tarantool during building of the package for the integration testing. At least, not so massively.

Reply:
Ok, fixed. Now testing is off (https://github.com/tarantool/tarantool/actions/runs/2578544828).